### ISSUE TITLE
Latest hashcat beta is not working for me

### ISSUE THREAD

**Issue by braindamagedjustnow** [hashcat/hashcat#3154]
Latest hashcat beta is not working for me:

```
hashcat (v6.2.5-195-g829396409) starting in benchmark mode

Benchmarking uses hand-optimized kernel code by default.
You can use it in your cracking session by setting the -O option.
Note: Using optimized kernel code limits the maximum supported password length.
To disable the optimized kernel code in benchmark mode, use the -w option.

hipDeviceGetAttribute(): 1


HIP API (HIP 4.4)
=================
* Device #1: AMD Radeon RX 6900 XT, skipped

OpenCL API (OpenCL 2.1 AMD-APP (3354.13)) - Platform #1 [Advanced Micro Devices, Inc.]
======================================================================================
* Device #5: AMD Radeon RX 6900 XT, 16256/16368 MB (13912 MB allocatable), 40MCU

Benchmark relevant options:
===========================
* --optimized-kernel-enable

-------------------
* Hash-Mode 0 (MD5)
-------------------

clCompileProgram(): CL_COMPILE_PROGRAM_FAILURE

C:\Users\User\AppData\Local\Temp\comgr-42291f\input\CompileSource:10:10: fatal error: '\"OpenCL\"/inc_vendor.h' file not found
#include STR(INCLUDE_PATH/inc_vendor.h)
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\Users\User\AppData\Local\Temp\comgr-42291f\input\CompileSource:7:16: note: expanded from macro 'STR'
#define STR(x) XSTR(x)
               ^~~~~~~
C:\Users\User\AppData\Local\Temp\comgr-42291f\input\CompileSource:6:17: note: expanded from macro 'XSTR'
#define XSTR(x) #x
                ^~
<scratch space>:2:1: note: expanded from here
"\"OpenCL\"/inc_vendor.h"
^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
Error: Failed to compile opencl source (from CL or HIP source to LLVM IR).

* Device #5: Kernel ./OpenCL/shared.cl build failed.

```
Windows 10 with latest AMD drivers

**Comment by matrix** [comment]
hi, try this patch:

```
diff --git a/src/backend.c b/src/backend.c
index 0aa098130..a95ac6286 100644
--- a/src/backend.c
+++ b/src/backend.c
@@ -8062,7 +8062,7 @@ int backend_session_begin (hashcat_ctx_t *hashcat_ctx)
       // tested on windows, linux, apple intel, apple silicon
       // when is builded with cygwin and msys, cpath_real doesn't work
       #if defined (_WIN) || defined (__CYGWIN__) || defined (__MSYS__)
-      build_options_len += snprintf (build_options_buf + build_options_len, build_options_sz - build_options_len, "-D KERNEL_STATIC -D INCLUDE_PATH=\"%s\" ", "OpenCL");
+      build_options_len += snprintf (build_options_buf + build_options_len, build_options_sz - build_options_len, "-D KERNEL_STATIC -D INCLUDE_PATH=%s ", "OpenCL");
       #else
       build_options_len += snprintf (build_options_buf + build_options_len, build_options_sz - build_options_len, "-D KERNEL_STATIC -D INCLUDE_PATH=\"%s\" ", folder_config->cpath_real);
       #endif
```

But you need get the last source from github, apply it and rebuild by yourself.

Let me know, thanks

**Comment by braindamagedjustnow** [comment]
Thank you matrix for the fast response!
I wish I can try it but I am not that advanced :) 
P.S.
Just saw you did a pull request that should resolve it. I will sit tight and wait to be merged 
Thank you one more time!

**Comment by matrix** [comment]
Anyway, if you want to try your hand ... https://github.com/hashcat/hashcat/blob/master/BUILD_WSL.md

Thanks ;)

**Comment by matrix** [comment]
@braindamagedjustnow new beta is up. Let us know if the problem is solved or not. Thanks :)

**Comment by braindamagedjustnow** [comment]
Still not working for me
```
hashcat (v6.2.5-205-gbe75e4b4e) starting in benchmark mode

Benchmarking uses hand-optimized kernel code by default.
You can use it in your cracking session by setting the -O option.
Note: Using optimized kernel code limits the maximum supported password length.
To disable the optimized kernel code in benchmark mode, use the -w option.

hipDeviceGetAttribute(): 1

HIP API (HIP 4.4)
=================
* Device #1: AMD Radeon RX 6900 XT, skipped

OpenCL API (OpenCL 2.1 AMD-APP (3354.13)) - Platform #1 [Advanced Micro Devices, Inc.]
======================================================================================
* Device #5: AMD Radeon RX 6900 XT, 16256/16368 MB (13912 MB allocatable), 40MCU

Benchmark relevant options:
===========================
* --optimized-kernel-enable

-------------------
* Hash-Mode 0 (MD5)
-------------------

clCompileProgram(): CL_COMPILE_PROGRAM_FAILURE

C:\Users\User\AppData\Local\Temp\comgr-725d50\input\CompileSource:7:10: fatal error: 'OpenCL/inc_vendor.h' file not found
#include M2S(INCLUDE_PATH/inc_vendor.h)
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<built-in>:4:16: note: expanded from here
#define M2S(x) XM2S(x)
               ^~~~~~~
<built-in>:3:17: note: expanded from here
#define XM2S(x) #x
                ^~
<scratch space>:2:1: note: expanded from here
"OpenCL/inc_vendor.h"
^~~~~~~~~~~~~~~~~~~~~
1 error generated.
Error: Failed to compile opencl source (from CL or HIP source to LLVM IR).

* Device #5: Kernel ./OpenCL/shared.cl build failed.
```

**Comment by matrix** [comment]
please post "hashcat -I" output
Thanks

**Comment by braindamagedjustnow** [comment]
```
hashcat (v6.2.5-205-gbe75e4b4e) starting in backend information mode

hipDeviceGetAttribute(): 1

hipDeviceGetAttribute(): 1

hipDeviceGetAttribute(): 1

hipDeviceGetAttribute(): 1

System Info:
============

OS.Name......: Windows
OS.Release...: N/A
HW.Platform..: N/A
HW.Model.....: N/A

HIP Info:
=========

HIP.Version.: 4.4

Backend Device ID #1
  Name...........: AMD Radeon RX 6900 XT
  Processor(s)...: 0
  Clock..........: 0
  Memory.Total...: 0 MB
  Memory.Free....: 0 MB
  Local.Memory...: 0 KB
  PCI.Addr.BDFe..: 0000:00:00.0

Backend Device ID #2
  Name...........: AMD Radeon RX 6900 XT
  Processor(s)...: 0
  Clock..........: 0
  Memory.Total...: 0 MB
  Memory.Free....: 0 MB
  Local.Memory...: 0 KB
  PCI.Addr.BDFe..: 0000:00:00.0

Backend Device ID #3
  Name...........: AMD Radeon RX 6900 XT
  Processor(s)...: 0
  Clock..........: 0
  Memory.Total...: 0 MB
  Memory.Free....: 0 MB
  Local.Memory...: 0 KB
  PCI.Addr.BDFe..: 0000:00:00.0

Backend Device ID #4
  Name...........: AMD Radeon RX 6900 XT
  Processor(s)...: 0
  Clock..........: 0
  Memory.Total...: 0 MB
  Memory.Free....: 0 MB
  Local.Memory...: 0 KB
  PCI.Addr.BDFe..: 0000:00:00.0

OpenCL Info:
============

OpenCL Platform ID #1
  Vendor..: Advanced Micro Devices, Inc.
  Name....: AMD Accelerated Parallel Processing
  Version.: OpenCL 2.1 AMD-APP (3354.13)

  Backend Device ID #5
    Type...........: GPU
    Vendor.ID......: 1
    Vendor.........: Advanced Micro Devices, Inc.
    Name...........: AMD Radeon RX 6900 XT
    Version........: OpenCL 2.0 AMD-APP (3354.13)
    Processor(s)...: 40
    Clock..........: 2015
    Memory.Total...: 16368 MB (limited to 13912 MB allocatable in one block)
    Memory.Free....: 16256 MB
    Local.Memory...: 64 KB
    OpenCL.Version.: OpenCL C 2.0
    Driver.Version.: 3354.13 (PAL,LC)
    PCI.Addr.BDF...: 36:00.0

  Backend Device ID #6
    Type...........: GPU
    Vendor.ID......: 1
    Vendor.........: Advanced Micro Devices, Inc.
    Name...........: AMD Radeon RX 6900 XT
    Version........: OpenCL 2.0 AMD-APP (3354.13)
    Processor(s)...: 40
    Clock..........: 2015
    Memory.Total...: 16368 MB (limited to 13912 MB allocatable in one block)
    Memory.Free....: 16256 MB
    Local.Memory...: 64 KB
    OpenCL.Version.: OpenCL C 2.0
    Driver.Version.: 3354.13 (PAL,LC)
    PCI.Addr.BDF...: 26:00.0

  Backend Device ID #7
    Type...........: GPU
    Vendor.ID......: 1
    Vendor.........: Advanced Micro Devices, Inc.
    Name...........: AMD Radeon RX 6900 XT
    Version........: OpenCL 2.0 AMD-APP (3354.13)
    Processor(s)...: 40
    Clock..........: 2015
    Memory.Total...: 16368 MB (limited to 13912 MB allocatable in one block)
    Memory.Free....: 16256 MB
    Local.Memory...: 64 KB
    OpenCL.Version.: OpenCL C 2.0
    Driver.Version.: 3354.13 (PAL,LC)
    PCI.Addr.BDF...: 39:00.0

  Backend Device ID #8
    Type...........: GPU
    Vendor.ID......: 1
    Vendor.........: Advanced Micro Devices, Inc.
    Name...........: AMD Radeon RX 6900 XT
    Version........: OpenCL 2.0 AMD-APP (3354.13)
    Processor(s)...: 40
    Clock..........: 2015
    Memory.Total...: 16368 MB (limited to 13912 MB allocatable in one block)
    Memory.Free....: 16256 MB
    Local.Memory...: 64 KB
    OpenCL.Version.: OpenCL C 2.0
    Driver.Version.: 3354.13 (PAL,LC)
    PCI.Addr.BDF...: 33:00.0
```

**Comment by matrix** [comment]
solved by #3186
